views/stockmetricspurchases.blade.php

```diff
@@ -0,0 +1,121 @@
+@extends('layout.default')
+
+@section('title', $__t('Stock Metrics: Purchases'))
+@section('activeNav', 'stockmetricspurchases')
+@section('viewJsName', 'metrics')
```

Comment:
`viewJsName` should be named like the view, so `stockmetricspurchases`.

Reply:
Got it. Easy fix.